README.md

```diff
@@ -99,6 +96,37 @@ For snapshot integrations use:
   </repository>
 ```
 
+and then include the desired `geomesa-*` dependencies:
```

Comment:
@metasim I think this block needs to go into the build/README.md; otherwise, it'd get clobbered later...